bbc/simorgh
# WSTEAM1-453: Add ATI tracking to live page

Resolves [WSTEAM1-453](https://jira.dev.bbc.co.uk/browse/WSTEAM1-453)

Overall changes
======
Adds ATI tracking to live pages, sample network message:  
```s: 598343
idclient: 109fb2d9-54a1-422d-adec-8494ae7b2018
s2: 70
p: live_coverage.c07zr0zwjnnt.page
r: 1920x1080x24x24
re: 1300x938
hl: 11x39x31
lng: en-GB
x1: [urn:bbc:tipo:topic:c07zr0zwjnnt]
x2: [responsive]
x3: [news-pidgin]
x4: [pcm]
x5: [http%3A%2F%2Flocalhost%3A7081%2Fpidgin%2Flive%2Fc07zr0zwjnnt]
x6: [http%3A%2F%2Flocalhost%3A7081%2Fpidgin%2Flive%2Fc07zr0zwjnnt]
x7: [live-coverage]
x8: [simorgh]
x9: [Live%20experience%20pagination]
ref: http://localhost:7081/pidgin/live/c07zr0zwjnnt
```

Code changes
======

- _Adds ATI Tracking component to `/ws-nextjs-app/pages/[service]/live/[id]/LivePageLayout.tsx`._

Testing
======
1. _Go to `/pidgin/live/cdvxv61l6x8t`_
2. Go to Network Tab. 
3. Check that ATI Payload is being sent.

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
